CHANGES.rst

```diff
@@ -5,6 +5,9 @@
 4.7.3 (unreleased)
 ==================
 
+- Fix persistency bug in the Python version
+  (`#118 https://github.com/zopefoundation/BTrees/issues/118`_).
```

Comment:
This is invalid ReST syntax, which gets caught by our Mac OS jobs in Travis CI running `twine check`:

```
   `long_description` has syntax errors in markup and would not be rendered on PyPI.
    line 39: Error: Unknown target name: "#118 https://github.com/zopefoundation/btrees/issues/118".
```

You're missing the `<` `>` around the URL.